### ISSUE TITLE
Adding a free software license

### ISSUE THREAD

**Issue by bzg** [captcha-com/laravel-captcha#10]
It would be nice to be able to reuse this source code.

Did you consider [adding a free software license](https://www.gnu.org/licenses/license-list.en.html)?

**Comment by jasperf** [comment]
Yes, and an MIT license like for most Laravel packages would be great. But either way we do need a license to know what the usage rights are. Now without a license all rights are reserved and the package is not open source:

_If a repository has no license, then all rights are reserved and it is not Open Source or Free. You cannot modify or redistribute this code without explicit permission from the copyright holder._

https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license?newreg=096bb4b8b0ff4d7f964da84ddf8d0856

_You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work._

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/licensing-a-repository#choosing-the-right-license
